charts/kyverno/values.yaml

```diff
@@ -44,7 +44,10 @@ global:
   nodeSelector: {}
 
   # -- Global List of node taints to tolerate. Non-global values will override the global value.
-  tolerations: []
+  tolerations:
+  - key: env
+    operator: Exists
+    effect: NoSchedule
```

Comment:
Adding it here will also affect main Kyverno pods. You may want to add as a separate configuration for test pods.